### PR TITLE
Add readme and update description

### PR DESCRIPTION
**Pull request by dominik-zeglen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+![Macaw-UI Logo](/stories/assets/macaw-ui-logo.svg)
+
+Welcome to the MacawUI.
+
+This is an official React UI Kit for Saleor, a GraphQL-First e-commerce platform for perfectionists.
+
+You can find most of the elements used in the creation of Saleor’s dashboard interface and use it to create projects for your specific purposes. Take a look a our [storybook docs](macaw-ui.vercel.app) for guides, component overview and project ideas.
```
Comment (kamilpastuszka) [inline README.md:7]: storybook docs link doesn't work

**Review by kamilpastuszka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dominik-zeglen) [reply README.md:7]: Oh nice catch

**Review by orzechdev — APPROVED** [review]

**Review by kamilpastuszka — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (karolkielecki) [inline README.md]:
```suggestion
You can find most of the elements used in the creation of Saleor's dashboard interface and use it to create projects for your specific purposes. Take a look at our [storybook docs](macaw-ui.vercel.app). You can find guides, component overview, and project ideas there.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Have a great time working on your projects and empowering your users.
```
Comment (karolkielecki) [inline README.md:9]:
```suggestion
Have a great time working on your projects and empowering your users. If you have any questions, feel free to let us know on [GitHub Discussions](https://github.com/mirumee/saleor/discussions). 
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+![Macaw-UI Logo](/stories/assets/macaw-ui-logo.svg)
+
+Welcome to the MacawUI.
+
+This is an official React UI Kit for Saleor, a GraphQL-First e-commerce platform for perfectionists.
```
Comment (karolkielecki) [inline README.md:5]:
```suggestion
This is the official React UI Kit for Saleor—an open-source GraphQL-first and Next.js ready e-commerce platform.
```

**Review by karolkielecki — CHANGES_REQUESTED** [review]

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**